azure-pipelines/integrationtests.yml

```diff
@@ -66,7 +66,7 @@ steps:
   - task: AzureKeyVault@1
     displayName: 'Azure Key Vault: SqlToolsSecretStore'
     inputs:
-      azureSubscription: 'ClientToolsInfra_670062 (88d5392f-a34f-4769-b405-f597fc533613)'
+      azureSubscription: 'STS Pipelines'

```

Comment:
Can we add a link to either the pipeline or the wiki page? Even a couple of guids... any for or  breadcrumb would help the next time around (and GUID are better than strings since they are harder do mutate).

Reply:
good idea! I added a comment on how to find this service connection. Since this is open source, I avoided putting a link to the actual pipeline